docs/.vitepress/config.ts

```diff
@@ -56,15 +56,24 @@ export default defineConfig({
       }, {
         text: 'Support',
         link: '/support',
-      }]
-    }],
+      }],
+      '/posts/': []
+    },
 
     socialLinks: [
       { icon: 'x', link: 'https://x.com/bashunit' },
       { icon: 'github', link: 'https://github.com/TypedDevs/bashunit' }
     ],
 
     nav: [{
+      text: '<strong>Docs</strong>',
+      link: '/quickstart',
+      activeMatch: '^/(?!posts)[^/]'
+    }, {
+      text: '<strong>Blog</strong>',
+      link: '/posts/',
```

Comment:
What do you think about using `/blog` instead of `/posts`, @Tito-Kati?

Reply:
@Chemaclass renamed 👍🏻